### PR TITLE
Bump version to 1.22.1 and update change logs 

### PR DESCRIPTION
**Pull request by dcoutts**
Also revert the recently-merged #2034 which accidentally synced some config files in the wrong direction.

### CODE REVIEW

**Review by deepfire — APPROVED** [review]
LGTM!

**Review by intricate — APPROVED** [review]

File: cardano-node/ChangeLog.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+  errors. Also added regression tests. This bug was introduced in the
+  unreleased 1.22.0 so has not been in any released version. (#1995)
+- Restore build compatibility with GHC 8.6.5, so this release builds with
+  both GHC 8.6.5 and 8.10.2. Future releases will support 8.10.x only. (#1956)
+- Further development of the multi-asset ledger rules (#1905, #1938, #1931)
+
+### consensus changes
+
+### network changes
+- Internal API improvements that will allow
```
Comment (intricate) [inline cardano-node/ChangeLog.md:18]: Oops, almost missed this. Incomplete sentence here.

Comment (dcoutts) [reply cardano-node/ChangeLog.md]: Well spotted. I had unrecorded changes.

**Comment by dcoutts** [comment]
bors merge

**Pull request merged**